src/Aspire.Hosting.AWS/AWSProvisioningException.cs

```diff
@@ -8,6 +8,5 @@ namespace Aspire.Hosting.AWS;
 /// </summary>
 /// <param name="message"></param>
 /// <param name="innerException"></param>
-public class AWSProvisioningException(string message, Exception? innerException = null) : Exception(message, innerException)
-{
-}
+public class AWSProvisioningException(string message, Exception? innerException = null)
```

Comment:
Can you revert any unnecessary whitespace changes?

Reply:
Yes OK. I thought whitespace would be useful.

Reply:
No, in general unnecessary changes shouldn't be made in PRs. A single PR should stick to a single set of changes. Making unnecessary changes not only makes the reviewer do more work, but it also clutter source control history.

Reply:
I understood you. Sorry for the mistake. I'll know for the future.